Xamarin.Forms.Platform.iOS/IVisualElementRenderer.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Xamarin.Forms.Platform.iOS
 {
-	public interface IVisualElementRenderer : IDisposable, IRegisterable
+	public interface IVisualElementRenderer : IDisposable, IRegisterable, IEffectControlProvider
```

Comment:
because if you made a renderer using the IVisualElementRenderer directly, your code would no longer compile with this change.